core/parsigex/parsigex.go

```diff
@@ -50,11 +56,12 @@ func NewParSigEx(tcpNode host.Host, sendFunc p2p.SendFunc, peerIdx int, peers []
 // ParSigEx exchanges partially signed duty data sets.
 // It ensures that all partial signatures are persisted by all peers.
 type ParSigEx struct {
-	tcpNode  host.Host
-	sendFunc p2p.SendFunc
-	peerIdx  int
-	peers    []peer.ID
-	subs     []func(context.Context, core.Duty, core.ParSignedDataSet) error
+	tcpNode    host.Host
+	sendFunc   p2p.SendFunc
+	peerIdx    int
+	peers      []peer.ID
+	verifyFunc func(context.Context, core.PubKey, core.Duty, core.ParSignedData) error
```

Comment:
nit: swap duty and pubkey, as that is the standard order we use in the code base